.github/workflows/ci.yml

```diff
@@ -78,7 +78,8 @@ jobs:
       run: |
         cabal test all
 
-    - name: Documentation
+    - if: matrix.ghc != '8.4.4'
+      name: Documentation
```

Comment:
Apparently, Haddock doesn't work on GHC 8.4. I decided to disable it just for this GHC version instead of completely removing building Haddock from CI or dropping GHC 8.4

![image](https://user-images.githubusercontent.com/4276606/140820352-f3bda07d-9d46-4350-a961-701debc951ab.png)

Reply:
What? 🤔